src/backend/distributed/utils/node_metadata.c

```diff
@@ -549,7 +550,7 @@ PrimaryNodeForGroup(int32 groupId, bool *groupContainsNodes)
  * given node.
  */
 static int
-ActivateNode(char *nodeName, int nodePort)
+ActivateNode(char *nodeName, int nodePort, int nodeId)
```

Comment:
Minor: `ActivateNode ()` doesn't use `nodeId` so we can undo this change

Reply:
using it again.